go.mod

```diff
@@ -4,9 +4,17 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
+	github.com/gogo/protobuf v1.2.1 // indirect
```

Comment:
are these needed? If I manually remove them they don't come back.

Reply:
```
go mod why -m github.com/gogo/protobuf
# github.com/gogo/protobuf
github.com/projectriff/system/cmd/managers/build
k8s.io/apimachinery/pkg/runtime
github.com/gogo/protobuf/proto
```

Leaving this as is for now, we can revisit if this seems wrong